app/api/flow_endpoint.py

```diff
@@ -283,7 +283,7 @@ async def debug_flow(flow: GraphFlow):
 
         ux = []
 
-        debug_info, log_list, event = await workflow.invoke(flow, event, ux, debug=True)
+        debug_info, log_list, event = await workflow.invoke(flow, event, debug=True)
```

Comment:
This will cause problems. Ux is needed please test it with the new version of tracardi.

Reply:
I added it.